### PR TITLE
V2.0

### LINKED ISSUE #10: ASE files from the Design language should be mastered here
The ASE files are the "single source of truth" for what the colors should be. We should keep them in a more visible place, Like here!

http://www.ibm.com/design/language/resources

### PR DESCRIPTION
**Pull request by seejamescode**
Would close #10, #11, #26

### CODE REVIEW

File: source/colors.js [hunk 1]
```diff
@@ -15,106 +15,108 @@ module.exports = {
...
+        { tone: '80', value: '#1d3458' },
         { tone: '90', value: '#19273c' }
       ]
     },
     {
-      name: 'warm blue',
+      name: 'cerulean',
+      synonyms: 'warm blue',
       values: [
         { tone: 'core', value: '#009de2' },
```
Comment (briandrum) [inline source/colors.js]: Shouldn't this be `#009bef `?

File: source/colors.js [hunk 2]
```diff
@@ -15,255 +15,354 @@ module.exports = {
...
-        { tone: '20', value: '#6eedd8' },
-        { tone: '30', value: '#41d6c3' },
-        { tone: '40', value: '#00b4a0' },
-        { tone: '50', value: '#008571' },
-        { tone: '60', value: '#006d5d' },
-        { tone: '70', value: '#005448' },
-        { tone: '80', value: '#003c32' },
-        { tone: '90', value: '#012b22' },
-        { tone: '100', value: '#000202' }
+        { tone: 'core', value: '#00A78F' },
```
Comment (briandrum) [inline source/colors.js]: Lower-case all `teal` values for consistency and efficiency (http://www.websiteoptimization.com/speed/tweak/lowercase/)?

File: source/colors.js (hunk 1 above)
Comment (seejamescode) [inline source/colors.js]: This looks like a bigger issue across multiple colors of which tone is considered core. I will look into it.

File: source/colors.js (hunk 2 above)
Comment (seejamescode) [inline source/colors.js]: Thanks for the catch!

File: source/colors.js (hunk 1 above)
Comment (seejamescode) [inline source/colors.js]: You are correct!

File: source/colors.js [hunk 3]
```diff
@@ -15,255 +15,354 @@ module.exports = {
...
+        { tone: '60', value: '#9320a2' },
+        { tone: '70', value: '#71237c' },
+        { tone: '80', value: '#501e58' },
+        { tone: '90', value: '#3b1a40' }
+      ]
+    },
+    {
+      name: 'violet',
+      values: [
+        { tone: 'core', value: '#7d24c9' },
```
Comment (briandrum) [inline source/colors.js]: Should this be `#7732bb `?

File: source/colors.js [hunk 4]
```diff
@@ -15,255 +15,354 @@ module.exports = {
...
-        { tone: '90', value: '#1a1314' },
-        { tone: '100', value: '#030000' }
+        { tone: '60', value: '#5f5757' },
+        { tone: '70', value: '#4b4545' },
+        { tone: '80', value: '#373232' },
+        { tone: '90', value: '#2a2626' }
       ]
     },
     {
       name: 'neutral-white',
```
Comment (briandrum) [inline source/colors.js]: De-hyphenate as in https://github.com/IBM-Design/colors/pull/27/files#diff-6a853a1f7639b0af7520dddefa30c2ffR292

File: source/colors.js [hunk 5]
```diff
@@ -15,255 +15,354 @@ module.exports = {
...
-        { tone: '40', value: '#ececec' }
+        { tone: 'core', value: '#fcfcfc' },
+        { tone: '1', value: '#fcfcfc' },
+        { tone: '2', value: '#f9f9f9' },
+        { tone: '3', value: '#f6f6f6' },
+        { tone: '4', value: '#f3f3f3' }
       ]
     },
     {
       name: 'cool-white',
```
Comment (briandrum) [inline source/colors.js]: De-hyphenate as in https://github.com/IBM-Design/colors/pull/27/files#diff-6a853a1f7639b0af7520dddefa30c2ffR292

File: source/colors.js [hunk 6]
```diff
@@ -15,255 +15,354 @@ module.exports = {
...
-        { tone: '30', value: '#f0f2f4' },
-        { tone: '40', value: '#ecf0f2' }
+        { tone: '1', value: '#fbfcfc' },
+        { tone: '2', value: '#f8fafa' },
+        { tone: '3', value: '#f4f7f7' },
+        { tone: '4', value: '#f0f4f4' }
       ]
     },
     {
       name: 'warm-white',
```
Comment (briandrum) [inline source/colors.js]: De-hyphenate as in https://github.com/IBM-Design/colors/pull/27/files#diff-6a853a1f7639b0af7520dddefa30c2ffR292

File: source/colors.js (hunk 4 above)
Comment (seejamescode) [inline source/colors.js]: What is the benefit you are look for here? Is it for the designer applications?

These de-hyphenations would be a breaking change for devs. Granted v2.0.0 allows for breaking changes, but we want to keep them at a low. Please make an issue if you still believe in this!

File: source/colors.js (hunk 6 above)
Comment (seejamescode) [inline source/colors.js]: See https://github.com/IBM-Design/colors/pull/27#discussion_r90673768

File: source/colors.js (hunk 5 above)
Comment (seejamescode) [inline source/colors.js]: See https://github.com/IBM-Design/colors/pull/27#discussion_r90673768

File: source/colors.js (hunk 3 above)
Comment (seejamescode) [inline source/colors.js]: I believe `#71237c`, but let me check

Comment (briandrum) [inline source/colors.js]: Perhaps I'm misunderstanding then: are the `core` values not always equal to one of the `tone`s?

File: source/colors.js (hunk 4 above)
Comment (seejamescode) [inline source/colors.js]: Oh snap, the un-hyphenated ones are the real problem then!

File: source/colors.js (hunk 3 above)
Comment (briandrum) [inline source/colors.js]: `#71237c ` is `purple-70`. This line would be for `violet`.

Comment (seejamescode) [inline source/colors.js]: Naw, they actually are a variety of tones that look best together. If you choose aaaall of the colors from one tone, it is not an amazingly appealing color wheel. I had just discovered this within the past week also. I believe our documentation should reflect that more.

Comment (seejamescode) [inline source/colors.js]: Core is not all 60, but you are correct: Violet Core is Violet 60 (#7732BB).

**Pull request merged**